### PR TITLE
[11.0][MIG] auth_totp_password_security

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: auth_totp_password_security/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
```
Comment (simahawk) [inline auth_totp_password_security/README.rst]: png

File: auth_totp_password_security/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
+
+Contributors
+------------
+
+* Oleg Bulkin <obulkin@laslabs.com>
```
Comment (simahawk) [inline auth_totp_password_security/README.rst:44]: Add "Do not contact contributors directly about support or help with technical issues."
See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: auth_totp_password_security/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    'name': 'MFA and Password Security Compatibility',
+    'summary': 'auth_totp and password_security compatibility',
+    'version': '11.0.1.0.0',
+    'category': 'Hidden',
+    'website': 'https://github.com/OCA/server-auth',
+    'author': 'LasLabs, Odoo Community Association (OCA)',
+    'license': 'LGPL-3',
+    'application': False,
```
Comment (simahawk) [inline auth_totp_password_security/__manifest__.py]: you can drop these flags, they are all defaults

File: auth_totp_password_security/controllers/main.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    @route()
+    def mfa_login_post(self, *args, **kwargs):
+        """Overload to check password expiration after MFA login"""
+        super_object = super(AuthTotpPasswordSecurity, self)
+        response = super_object.mfa_login_post(*args, **kwargs)
+
+        if not request.params.get('login_success'):
+            return response
+
+        user = request.env['res.users'].sudo().browse(request.uid)
```
Comment (simahawk) [inline auth_totp_password_security/controllers/main.py]: you have `request.env.user`

**Review by simahawk — CHANGES_REQUESTED** [review]
LGTM, just small things

File: auth_totp_password_security/README.rst (hunk 2 above)
Comment (MiquelRForgeFlow) [reply auth_totp_password_security/README.rst:44]: Done

**Comment by pedrobaeza** [comment]
Please rebase and remove the other PR reference

File: auth_totp_password_security/controllers/main.py [hunk 5]
```diff
@@ -16,7 +15,7 @@ def mfa_login_post(self, *args, **kwargs):
         if not request.params.get('login_success'):
             return response
 
-        user = request.env['res.users'].sudo().browse(request.uid)
+        user = request.env.user
```
Comment (JordiBForgeFlow) [inline auth_totp_password_security/controllers/main.py]: Change to request.env.user.sudo(request.uid)

**Comment by MiquelRForgeFlow** [comment]
@jbeficent @pedrobaeza ready

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional review.

**Pull request merged**